service-loadbalancer/service-loadbalancer.dot

```diff
@@ -0,0 +1,26 @@
+digraph serviceloadbalancer {
+
+   subgraph cluster_0 {
+        user->haproxy [label="Old way of doing \n bare-metal load balancing: \nrequires NodePorts"]
+        user->haproxyLB [label="Service Loadbalancer: Pluggable + pod-aware loadbalancing alternative\n which forwards directly to endpoints."];
+
+} 
+        subgraph cluster_a { 
+        label="Nodeports";
+         haproxy->k1nodeport_8001;
+         haproxy->k2nodeport_8001;
+         k1nodeport_8001->service;
+         k2nodeport_8001->service;
+         service->k1PostgreSQL [label=targetPort_80];
```

Comment:
nit.  possibly this could be **outside** the nodeport box?    just a thought I'd put in here as  part of a self-review im doing.